diff --git a/src/kcompat.h b/src/kcompat.h
index 21e9818..97abc2f 100644
--- a/src/kcompat.h
+++ b/src/kcompat.h
@@ -7074,7 +7074,8 @@ devlink_flash_update_status_notify(struct devlink __always_unused *devlink,
 /*****************************************************************************/
 #if (LINUX_VERSION_CODE < KERNEL_VERSION(5,4,0))
 #if (!(RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(8,2)) && \
-     !(SLE_VERSION_CODE >= SLE_VERSION(15,2,0)))
+     !(SLE_VERSION_CODE >= SLE_VERSION(15,2,0)) && \
+     !(UBUNTU_VERSION_CODE >= UBUNTU_VERSION(4,15,0,159)))
 static inline unsigned int skb_frag_off(const skb_frag_t *frag)
 {
        return frag->page_offset;
